data/mods/Aftershock/mobs/robots.json

```
@@ -785,5 +785,65 @@
       "STUN_IMMUNE"
     ],
     "armor": { "bash": 12, "cut": 8, "bullet": 6 }
+  },
+  {
+    "id": "mon_mech_horse",
+    "type": "MONSTER",
+    "name": { "str": "STEED" },
+    "description": "Found on nearly every frontier planet, the ubiquitous TSKBEM STEED (Surface Traversing Equine Emulating Drone) is an incredibly rugged quadrapedal machine based on old Earth horses and mules.  Capable of crossing the roughest terrain, powered by a compact atomic reactor, generous storage capacity, light armor, and a lifetime warranty, the STEED is a priceless tool for prospectors and homesteaders.  This version has been ruggedized against Salus IV's extreme cold, but does not keep you warm while riding.",
```

```suggestion
    "description": "Found on nearly every frontier planet, the ubiquitous TSKBEM STEED (Surface Traversing Equine Emulating Drone) is an incredibly rugged quadrupedal machine based on old Earth horses and mules.  Capable of crossing the roughest terrain, powered by a compact atomic reactor, generous storage capacity, light armor, and a lifetime warranty, the STEED is a priceless tool for prospectors and homesteaders.  This version has been ruggedized against Salus IV's extreme cold, but does not keep you warm while riding.",
```

"ruggedized" should be added to the dictionary, though.